src/lib/utils/jsonResponses/Forms.ts

```diff
@@ -0,0 +1,50 @@
+//https://docs.synapse.org/rest/org/sagebionetworks/repo/model/form/FormGroup.html
+export type FormGroup = {
+  groupId?: string
```

Comment:
is name the only guaranteed field for FormGroup?

Reply:
That's a good point.  In other service calls, objects are used for both request and response objects (where fields like "id" would not be filled in for the request), but that's not applicable for this.  I'll mark other fields as required...